src/modules/Kdump.rb

```diff
@@ -1020,7 +991,6 @@ def high_memory_supported?
     publish :variable => :proposal_valid, :type => "boolean"
     publish :variable => :available_partitions, :type => "list <string>"
```

Comment:
When we are removing the global variable we should remove this too. Does anyone still need it ?